src/components/InventoryTabs/ConventionalSystems/ConventionalSystemsTab.js

```diff
@@ -295,7 +295,6 @@ const ConventionalSystemsTab = ({
           ],
         }}
         bulkSelect={bulkSelectConfig}
-        onRowClick={(_e, id, app) => navigate(`/${id}${app ? `/${app}` : ''}`)}
```

Comment:
I'm not sure what this was meant to link to exactly. When the `onRowClick` is called it will receive an `event`, `id`, but the third argument is not an "app" or anything, but `isMetaKey` which determines if a certain keyboard key is pressed.

Reply:
Also, by default the inventory table will link/navigate to the systems details page in the inventory, like it did here in the passed in `onRowClick`. So this became obsolete.